### DISCUSSION TITLE
Allow running arch test in views

### DISCUSSION THREAD

**Discussion by andradewall** [pestphp/pest#971] [Ideas]
Hello.

Running the following example, got from [this section](https://pestphp.com/docs/arch-testing#content-tobeused), if we left a `dump()` function in any view, the test doesn't catch it.
```php
test('globals')
    ->expect(['dd', 'dump'])
    ->not->toBeUsed();
```

As I'm not capable to try a PR for this feature yet, my suggestion is to be add a option to extend (by default or optionally) this test to the views also.

**Comment by Ahrengot** [comment]
Agreed. I have been looking through docs an issues trying to figure out if this was supported or not, as I couldn't get my tests to fail when i added a `@ray()` or `@dd()` to a view file, which I assume is fairly common practise.
